apps/frontend/src/app/Components/Artifact/ArtifactFilterDisplay.tsx

```diff
@@ -1,12 +1,13 @@
-import { faBan, faChartLine } from "@fortawesome/free-solid-svg-icons";
-import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { allArtifactSets, allSlotKeys } from "@genshin-optimizer/consts";
 import { BusinessCenter, Lock, LockOpen, PersonSearch } from '@mui/icons-material';
+import BlockIcon from '@mui/icons-material/Block';

```

Comment:
Is there a reason this isn't just importing `Block` from `@mui/icons-material`?

Reply:
Its how the mui Icon documentation lets you copy when you look up icons. so I was too lazy to change their convention, which kind of conflict with our existing import convention. I'd say importing `...Icon` is probably better since it explicitly states that this var is an icon.

Reply:
So at some point we should move all the icons-material imports to be like this? From what I can tell, bundle size is unaffected by this so its purely for us as devs. It would make the import sections a bit longer, but I guess its kinda whatever for it to be longer

Reply:
perhaps, but I think i have enough churn in this PR